### PR TITLE
 [GFA1] improve definition of C lines (after #58)

### PR DESCRIPTION
**Pull request by ggonnella**
This branch reconstructs the commits of the merge request #58 and applies them to the current master branch.

### CODE REVIEW

**Comment by ggonnella** [comment]
This PR closes issue #57

**Comment by sjackman** [comment]
This PR achieved quorum in PR #57.

File: GFA1.md [hunk 1]
```diff
@@ -132,19 +132,31 @@ The Overlap field is optional and can be `*`, meaning that the CIGAR string is n
 
 # `C` Containment line
 
-A containment line represents an overlap between two segments where one is contained in the other.
+A containment line represents an overlap between two segments where one (the `Container` segment)
+is contained in the other (the `Contained` segment). The `Pos` field stores the leftmost
```
Comment (sjackman) [inline GFA1.md]: I believe Container and Contained should be swapped in the following sentence.

> A containment line represents an overlap between two segments where one (the `Container` segment) is contained in the other (the `Contained` segment).

Comment (ggonnella) [inline GFA1.md]: Well spotted

**Comment by sjackman** [comment]
Merged. Thanks for your contribution to the GFA 1 spec, Giorgio!

**Pull request merged**